frontend/src/components/Forms/InputForm/index.tsx

```diff
@@ -24,6 +24,8 @@ export const InputForm = ({
       onSubmit={handleOnSubmit}
       initialValues={initialValues}
       validationSchema={validationSchema}
+      enableReinitialize={true}
```

Comment:
did we determine this was required for sure.

Reply:
removed it